src/router/index.js

```diff
@@ -4,6 +4,7 @@ import HomeView from '../views/HomeView.vue'
 import StudentsView from '../views/StudentsView.vue'
 import ExercisesView from '../views/ExercisesView.vue'
 import GradesView from '../views/GradesView.vue'
+import CoursesView from '@/views/CoursesView.vue'
```

Comment:
Just for consistency here we should use the same path (absolute or relative, but not mix them)

Reply:
Okay! Fixed on the next commit